lib/locomotive/steam/services/site_finder_service.rb

```diff
@@ -6,7 +6,7 @@ class SiteFinderService
       attr_accessor_initialize :repository, :request
 
       def find
-        repository.by_domain(request.host)
+        repository.by_domain(request.host) if request
```

Comment:
@did, I saw an issue in wagon when this conditional wasn't included. I'm not sure if wagon has a different concept of requests but this fixed the problem without any regressions in existing testing.